api4/channel_test.go

```diff
@@ -969,15 +969,6 @@ func TestGetAllChannels(t *testing.T) {
 	defer th.TearDown()
 	Client := th.Client
 
-	var originalConfigVal bool
-	th.App.UpdateConfig(func(cfg *model.Config) {
-		originalConfigVal = *cfg.TeamSettings.ExperimentalViewArchivedChannels
-		*cfg.TeamSettings.ExperimentalViewArchivedChannels = true
-	})
-	defer th.App.UpdateConfig(func(cfg *model.Config) {
-		*cfg.TeamSettings.ExperimentalViewArchivedChannels = originalConfigVal
-	})
-
```

Comment:
Maybe explicitly set to `false` to test that it doesn't affect the behaviour of these methods. The test will also make the intent and usage clearer.